content/post/2017-09-15-friday-hacks-137-Sep-15.md

```diff
@@ -0,0 +1,29 @@
+---
+title: "Friday Hacks #137, September 15"
+date: 2017-09-12 09:40:35.299431
+author: Herbert
+---
+
+This Friday, our very own coreteam members will be haring their internship experiences both locally and overseas. We will also invite NUS students who have done internships in various companies.
+
+{{% friday_hack_header venue="Seminar Room 1, NUS School of Computing (COM1-0206)" date="September 15" %}}
+
+### Internships, and How to Get Them
+
+#### Talk Description:
+
+Learn why you should do internships, how to choose, and of course, how to get them!
+
+Ask questions of other students who have interned locally and overseas at various companies. This includes overseas like Google, Facebook, Twitter, Asana, and local companies like Google Singapore, Twitter Singapore, Carousell, Garena, and Tinkerbox!
+
+To the freshmen, it's never too early to start!
+
+You've probably read about tech internships and all their various perks. Heck, you've probaby even seen a movie about it. Now's the time to live it. This talk will address all the questions you've ever had about internships and help you on your way to getting one.
+
+Free pizza is served before the talks.
+
+#### Speaker Profile
+
+Varun is a Year 4 Computer Science student who did internships at Grab, Garena, Asana, and Jump Trading.
+
+Advay is a Year 3 Computer Science student who did internships at Xfers and Twitter.
```

Comment:
I forgot if the NUS enterprise stuff needs to be added in the post

Reply:
We never did so far. :p we do on FB postings though.

Reply:
Can you change the order to Most recent first?